layers/cmd/ingest_csv.go

```diff
@@ -100,61 +100,71 @@ func (ci *CSVIngester) Run(pipeline *pipeline.PipelineContext) error {
 			pipeline.SetGraph(ls.NewDocumentGraph())
 		}
 		reader.Comma = rune(ci.Delimiter[0])
+		var doneErr error
+		func() {
```

Comment:
Move the func() inside the for-loop, so if one row fails, it continues with the reamining